src/styles.ts

```diff
@@ -85,13 +87,15 @@ const widgetBubbleStyle = {
 
 const widgetIframeStyle = {
   container: {
-    position: 'absolute',
+    position: 'fixed',
     // Size and position values are set in iframewrapper.ts
 
     display: 'flex',
     flexDirection: 'column',
     boxShadow: '4px 5px 4px rgba(0, 0, 0, 0.25)',
     borderRadius: '10px',
+
+    zIndex: '214748363',
```

Comment:
how did you find this is the max value of z-index?

Reply:
I found it in digital ocean blog in google. It displays ±2147483647